libraries/wallet/wallet.cpp

```diff
@@ -770,6 +770,15 @@ class wallet_api_impl
 
       return true;
    }
+
+    void quit()
+    {
+        ilog( "exiting wallet ..." );
+        save_wallet_file();
```

Comment:
What if this throws an exception?

Reply:
will ad try/catch then, right ? do you mean that ?